plugins/com.google.cloud.tools.eclipse.ui.util/META-INF/MANIFEST.MF

```diff
@@ -16,6 +16,7 @@ Import-Package: com.google.cloud.tools.eclipse.ui.util,
  com.google.cloud.tools.eclipse.util.status,
  com.google.common.annotations;version="[20.0.0,21.0.0)",
  com.google.common.base;version="[20.0.0,21.0.0)",
+ com.google.common.net;version="15.0.0",
```

Comment:
why 15.0.0? If this is part of Guava it should be 20.0.0, right?

Reply:
Oops!  Missed that.